src/container/oid_iv.c

```diff
@@ -130,17 +133,28 @@ oid_iv_ent_update(struct ds_iv_entry *ns_entry, struct ds_iv_key *iv_key,
 	d_rank_t		myrank = dss_self_rank();
 	int			rc;
 
+	if (src == NULL) {
```

Comment:
did you see such case that with NULL src in oid_iv_ent_update() in test? may come from oid_iv_invalidate()

Reply:
no, but i thought we can just add it for now to be sure